be/src/exec/pipeline/pipeline_driver.cpp

```diff
@@ -466,6 +460,27 @@ void PipelineDriver::_close_operators(RuntimeState* runtime_state) {
     }
 }
 
+void PipelineDriver::_set_operator_execute_mode(RuntimeState* state, MemTracker* tracker, OperatorPtr& op,
```

Comment:
this function name is quite confusing,  how about
adjust_spilling_chance_to_adapt_memory_usage.

Reply:
We may still call this function for some non-spillable operators. For example local-exchange, we might try to reduce the size of its intermediate buffer.
so maybe we could rename it to adjust_memory_usage?

Reply:
fixed